com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -1248,13 +1250,17 @@ class RenderDistortionPassData
                 builder.SetRenderFunc(
                 (RenderDistortionPassData data, RenderGraphContext context) =>
                 {
+                    if (data.smoothDistortion)
+                        HDUtils.BlitCameraTexture(context.cmd, m_CameraColorBuffer, data.sourceColorBuffer);
```

Comment:
This seems to be a mistake. Looks like it uses the regular path camera color buffer.
(And this will also GCAlloc here). I suppose it was meant to be passed along with the intermediate buffer in the pass data.

Reply:
fixed